Missing /usr/bin/dot using plantuml

I wrote a simple AsciiDoc document containing a plantuml diagram, but when I try to render it an exception is thrown:

```
$ docker run -it --rm -v ${PWD}:/documents rochdev/alpine-asciidoctor asciidoctor -r asciidoctor-diagram documentation.adoc
java.io.IOException: Cannot run program "/usr/bin/dot": error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at java.lang.Runtime.exec(Runtime.java:620)
	at net.sourceforge.plantuml.cucadiagram.dot.ProcessRunner$MainThread.startThreads(ProcessRunner.java:158)
	at net.sourceforge.plantuml.cucadiagram.dot.ProcessRunner$MainThread.runJob(ProcessRunner.java:118)
	at net.sourceforge.plantuml.api.TimeoutExecutor$MyThread.run(TimeoutExecutor.java:74)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 4 more
```

It looks like something is missing, maybe graphviz is not fully installed. It works fine with ditaa diagram.

Thank you.
Lorenzo

--- a/full/Dockerfile
+++ b/full/Dockerfile
@@ -6,7 +6,7 @@ ENV LIBRARY_PATH=/lib:/usr/lib
 ENV BACKENDS /asciidoctor-backends
 ENV GVM_AUTO_ANSWER true
 
-RUN apk add --update --no-cache gcc g++ make bash curl unzip tar openjdk8-jre ruby-dev ruby ruby-nokogiri ruby-json asciidoctor python python-dev py-setuptools py-pip ttf-dejavu libjpeg-turbo libjpeg-turbo-dev zlib zlib-dev zip && \
+RUN apk add --update --no-cache gcc g++ make bash curl unzip tar openjdk8-jre ruby-dev ruby ruby-nokogiri ruby-json asciidoctor python python-dev py-setuptools py-pip ttf-dejavu graphviz libjpeg-turbo libjpeg-turbo-dev zlib zlib-dev zip && \
     gem install --no-ri --no-rdoc asciidoctor-diagram && \
     gem install --no-ri --no-rdoc asciidoctor-pdf --pre && \
     gem install --no-ri --no-rdoc asciidoctor-epub3 --pre && \

--- a/plus/Dockerfile
+++ b/plus/Dockerfile
@@ -6,7 +6,7 @@ ENV LIBRARY_PATH=/lib:/usr/lib
 ENV BACKENDS /asciidoctor-backends
 ENV GVM_AUTO_ANSWER true
 
-RUN apk add --update --no-cache gcc g++ make bash curl unzip tar openjdk8-jre ruby-dev ruby ruby-nokogiri ruby-json asciidoctor python python-dev py-setuptools py-pip ttf-dejavu libjpeg-turbo libjpeg-turbo-dev zlib zlib-dev zip && \
+RUN apk add --update --no-cache gcc g++ make bash curl unzip tar openjdk8-jre ruby-dev ruby ruby-nokogiri ruby-json asciidoctor python python-dev py-setuptools py-pip ttf-dejavu graphviz libjpeg-turbo libjpeg-turbo-dev zlib zlib-dev zip && \
     apk add --update --no-cache cmake bison flex libffi-dev glib-dev pango-dev cairo-dev gdk-pixbuf-dev libxml2-dev && \
     gem install --no-ri --no-rdoc asciidoctor-diagram && \
     gem install --no-ri --no-rdoc asciidoctor-pdf --pre && \

